tutorials/autotvm/tune_simple_template.py

```diff
@@ -294,8 +296,9 @@ def matmul(N, L, M, dtype):
     builder='local',
     runner=autotvm.LocalRunner(number=5))
 
-# begin tuning, log records to file `matmul.log`
-tuner = autotvm.tuner.XGBTuner(task)
+# begin tuning with RandomTuner, log records to file `matmul.log`
```

Comment:
Capital B.
Also please be aware that this file will be processed to generate a tutorial web page, so all comments will be tutorial texts. Although it's annoying, you need to follow the specific comment rules. For example, a paragraph should start with a line with full of "#". You can simply copy line 31 here to make it work.

Reply:
![image](https://user-images.githubusercontent.com/535650/70854594-08764200-1e7b-11ea-823a-5e13d6739ebe.png)
I was hoping in-code comment would be sufficient, which won't break the section? Let me know if you still want to create explicit break in the code section :).

Reply:
I checked the current version on the website and I agree with you that in-code comment is better for this case.